samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -0,0 +1,163 @@
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+A simple application demonstrating the BLE Direction Finding CTE Broadcaster functionality
+by transmission of Constant Tone Extension with periodic advertising PDUs.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52833dk_nrf52833
+
+The sample also requires an antenna matrix to use it in Angle of Departure mode.
+It can be a Nordic Semiconductor design 12 patch antenna matrix or any other antenna matrix.
+
+Configuration
+*************
+
+|config|
+
+The sample supports two Direction Finding modes:
+
+* Angle of Arrival
+* Angle of Departure
+
+By default both modes are available in the sample application.
+
+Angle of Arrival mode
+=====================
+
+To build this sample with Angle of Arrival mode only, set :makevar:`OVERLAY_CONFIG`
+to :file:`overlay-aoa.conf`.
+
+See :ref:`cmake_options` for instructions on how to add this option.
+For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars`
+in the Zephyr documentation.
+
+Antenna matrix configuration
+============================
+
+To use this sample with Angle of Departure enabled, additional GPIOS configuration
+is required to enable control of the antenna array. Example of such configuration
+is provided in :file:`nrf52833dk_nrf52833.overlay`.
+
+:file:`nrf52833dk_nrf52833.overlay` is a device tree overlay that provides information
+about which GPIOS should be used by Radio peripheral to switch between antenna
+patches during CTE transmission in AoD mode. At least two GPIOs must be provided
+to enable antenna switching. GPIOS are used by Radio peripheral in order following
+indices of :code:`dfegpio#-gpios` properties. The order is important because it
+affects mapping of antenna switch patterns to GPIOS (see `Antenna patterns`_).
+
+To successfully use Direction Finding Beacon with AoD mode enabled provide
+following data related with owned antenna matrix design:
+
+* provide GPIO pins to :code:`dfegpio#-gpios` propreties in :file:`nrf52833dk_nrf52833.overlay` file
+* provide default antenna that will be used to transmit PDU :code:`dfe-pdu-ant`
+  property in :file:`nrf52833dk_nrf52833.overlay` file
+* update antenna patterns in :cpp:var:`ant_patterns` array in :file:`main.c`.
+
+Antenna patterns
+================
+
+The antenna switch pattern is a binary number where each bit is applied to
+a particular antenna GPIO pin. For example, the pattern 0x3 means that antenna
+GPIOs at index 0,1 will be set, and following are left unset.
+
+This also means that, for example, when using four GPIOs, the patterns count
+cannot be greater than 16 and maximum allowed value is 15.
+
+If the number of switch-sample periods is greater than the number of stored
+switch patterns, then the radio loops back first pattern.
```

Comment:
Loops back to the first pattern?

Reply:
Yes, that sounds better. Corrected

Reply:
And I also see that
switch pattern -> switching pattern

Must have missed it last time!

Reply:
Corrected